test/expect-tests/dune_file_watcher/dune

```diff
@@ -59,6 +59,7 @@
    (sandbox always)))
  (libraries
   base
+  dune_config
```

Comment:
why do you need this dependency?

Reply:
Here's a different approach:

1. We move the construction of the exclusion list outside the file watcher
2. The file watcher will instead take a predicate `(string -> bool)`.
3. The unit test will just use this predicate directly rather than accessing it through the file watcher

Reply:
That suggestion doesn't address your original question ("why do you need this dependency?"). In particular, if as suggested we move the built-in exclusion list outside of the file watcher, then there has to be an added `(libraries ...)` dependency in `test/expect-tests/dune_file_watcher/dune` to test those built-in exclusions.

I dropped the extra dependency and am just testing a slimmed-down copy of the built-in exclusions.

Reply:
Oh okay. I didn't realize the dependency was just on the test. This is fine.